iroh-resolver/src/racing.rs

```diff
@@ -1,5 +1,6 @@
-use crate::resolver::{
```

Comment:
do we need the racing loader at all anymore?

Reply:
no, it is fully integrated now

Reply:
nice. I vote we remove in this PR